modules/apps/foundation/portal-security-audit/portal-security-audit-api/src/main/java/com/liferay/portal/security/audit/configuration/AuditConfiguration.java

```diff
@@ -21,7 +21,7 @@
 /**
  * @author Michael C. Han
  */
-@ExtendedObjectClassDefinition(category = "security-general")
```

Comment:
I believe this is meant to be just `security`? Please revert my change if I'm wrong.